types/src/main.ts

```diff
@@ -1929,6 +1929,7 @@ export namespace LanguageKind {
 	export const Go	= 'go' as const;
 	export const Groovy	= 'groovy' as const;
 	export const Handlebars	= 'handlebars' as const;
+	export const Haskell	= 'haskell' as const;
```

Comment:
Needs @since and @proposed. See `D`

Reply:
The whole enum has `@since 3.18` and `@proposed`, so are the individual entries not all implicitly the same? It seems a bit odd that only a few of the entries have those annotations: shouldn't all of them have it or none of them?

Reply:
Good point.